src/services/member/plugins/profile/schemas.ts

```diff
@@ -1,66 +1,70 @@
-import { S } from 'fluent-json-schema';
+import { Type } from '@sinclair/typebox';
 import { StatusCodes } from 'http-status-codes';
 
 import { FastifySchema } from 'fastify';
 
-import { error, uuid } from '../../../../schemas/fluent-schema';
-import { partialMember } from '../../../item/schema';
+import { customType, registerSchemaAsRef } from '../../../../plugins/typebox';
+import { errorSchemaRef } from '../../../../schemas/global';
 
-const memberSharedSchema = S.object()
-  .additionalProperties(false)
-  .prop('bio', S.string())
-  .prop('facebookID', S.anyOf([S.string(), S.null()]))
-  .prop('linkedinID', S.anyOf([S.string(), S.null()]))
-  .prop('twitterID', S.anyOf([S.string(), S.null()]));
+const inputProfileMemberSchemaRef = registerSchemaAsRef(
+  'memberShared',
+  'Member Shared',
```

Comment:
What do you think of this naming ?
```suggestion
  'publicProfile',
  'Public Profile',
```